framework/Models/EnsembleModel.py

```diff
@@ -656,7 +656,7 @@ def _externalRun(self,inRun, jobHandler):
           while not moveOn:
             if jobHandler.availability() > 0:
               # run the model
-              self.modelsDictionary[modelIn]['Instance'].submit(originalInput[modelIn], samplerType, jobHandler, **copy.deepcopy(inputKwargs[modelIn]))
+              self.modelsDictionary[modelIn]['Instance'].submit(originalInput[modelIn], samplerType, jobHandler, **inputKwargs[modelIn])
```

Comment:
I believe I added this deepcopy, if it passes the tests without it, then this should be used instead. I may have been overly cautious when passing things across jobs/threads. This may also be able to be changed on the Steps.py file.  I will have to check that outside of this PR.